docs/decisions/0018-standarize-django-po-files.rst

```diff
@@ -0,0 +1,157 @@
+Standardize ``django.po`` and ``djangojs.po`` files
+===================================================
+
+Status
+------
+
+Accepted
+
+Context
+-------
+
+#. The edx-platform splits its translations files to ``mako-studio.po``,
+   ``django-partial.po`` among 11 files in total to make it possible for
+   translators to focus on learner-facing text
+   and de-prioritize educator-facing text. The idea was that LMS would be
+   translated into more languages than CMS (Studio) would.
```

Comment:
```suggestion
```suggestion
#. The edx-platform splits its translations into ``mako-studio.po``,
   ``django-partial.po``, and 9 other files.  This was done with the
   intention of supporting more languages in the LMS than in
   the CMS (Studio).
```

Reply:
```suggestion
#. The edx-platform splits its translations into ``mako-studio.po``,
   ``django-partial.po``, and 9 other files.  This was done with the
   intention of supporting more languages in the LMS than in
   the CMS (Studio).
```

Reply:
```suggestion
#. The edx-platform splits its translations into ``mako-studio.po``,
   ``django-partial.po``, and 9 other files.  This was done with the
   intention of supporting more languages in the LMS than in
   the CMS (Studio).
```